frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,35 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
+                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" required focus-if>
```

Comment:
there is no `name` attributes so `ng-message` can't show errors below input box in case of having one. See Line 13 validates value using `changepwd.isValid.old_password` (here `old_password` is referenced for the value of name attribute of input)
Add name attribute in all the input fields. See the reference in `signup/ reset password confirm`.